src/main/resources/db/changelog/changes/13-insert-tasks.yaml

```
@@ -0,0 +1,53 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-tasks
+      author: fakhrutdinov-eduard
+      changes:
+        - insert:
+            tableName: tasks
+            columns:
+              - column:
+                  name: name
+                  value: "TASK 1"
+              - column:
+                  name: description
+                  value: "Create user entity as described in README.md"
+              - column:
+                  name: priority
+                  value: "HIGH"
+              - column:
+                  name: status
+                  value: "IN_PROGRESS"
+              - column:
+                  name: due_date
+                  valueDate: "2024-06-05"
+              - column:
+                  name: project_id
+                  value: 1
```

It is better to do valueComputed here, ID could change.
```suggestion
                  valueComputed: (SELECT id FROM projects WHERE name = 'some name')
```